src/azure-cli/HISTORY.rst

```diff
@@ -11,6 +11,10 @@ Release History
 
 * Fix #10996: Polish error for `--force-change-password-next-login` in `az ad user update` when `--password` is not specified
 
+**ACR**
```

Comment:
@rosanch the release history is alphabetized by team. Please bring ACR to the top.